lib/tasks/create_application.rake

```diff
@@ -1,22 +1,39 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/BlockLength
 namespace :auth do
   desc 'Create an OAuth2 authorised client application'
   task create_client_application: :environment do
     require 'doorkeeper/orm/active_record/application'
+    require 'tty-prompt'
 
-    if ENV['NAME'].blank?
-      puts 'Usage: rake auth:create_client_application NAME=name'
-      exit
+    prompt = TTY::Prompt.new
+
+    name = prompt.ask("What's the application's name?")
+
+    scopes = %w[read write]
+    scope = prompt.multi_select("What's the scope of the #{name} application?", scopes)
+
+    application = Doorkeeper::Application.new(name: name, scopes: scope)
+
+    suppliers = Supplier.all.each_with_object({}) do |s, hsh|
+      hsh[s.name] = s.id
+    end.merge('No owner' => nil)
```

Comment:
`hsh` and `s` aren't meaningful names. Could this be refactored to (e.g.) `memo` and `supplier`?